.github/workflows/formatter.yaml

```diff
@@ -0,0 +1,29 @@
+name: Formatter
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+  pull_request:
+    branches:
+      - main
+      - dev
```

Comment:
We use 'master' branch, instead of 'main', replace please all main in github actions to master